cmd/kubeadm/app/cmd/alpha/kubeconfig.go

```diff
@@ -96,6 +105,7 @@ func newCmdUserKubeConfig(out io.Writer) *cobra.Command {
 	cmd.Flags().StringVar(&token, options.TokenStr, token, "The token that should be used as the authentication mechanism for this kubeconfig, instead of client certificates")
 	cmd.Flags().StringVar(&clientName, "client-name", clientName, "The name of user. It will be used as the CN if client certificates are created")
 	cmd.Flags().StringSliceVar(&organizations, "org", organizations, "The orgnizations of the client certificate. It will be used as the O if client certificates are created")
+	cmd.Flags().DurationVar(&validityPeriod, "validity-period", kubeadmconstants.CertificateValidity, "The validity period of the client certificate. It is an offset from the current time.")
```

Comment:
Default is 1 year.
Need we add the default value in the flag description?

Reply:
It seems that `cobra` will automatically display the default value.
```sh
# kubeadm kubeconfig user -h
Flags:
          --validity-period duration   The validity period of the client certificate. It is an offset from the current time. (default 8760h0m0s)
```